charts/prometheus/values.yaml

```
@@ -592,8 +592,10 @@ server:
     # volumeName: ""
 
   emptyDir:
-    ## Prometheus server emptyDir volume size limit
+    ## Prometheus server emptyDir volume
+    ## Configure size limit and medium memory
     ##
+    mediumMemory: false
```

I'd suggest to keep the field name, set to an empty string by default.
```suggestion
    medium: ""
```